features/admin.actions.v1/components/pre-update-password-action-config-form.tsx

```diff
@@ -316,114 +352,128 @@ const PreUpdatePasswordActionConfigForm: FunctionComponent<PreUpdatePasswordActi
                     actionTypeApiPath={ actionTypeApiPath }
                     actionId={ initialValues?.id }
                 />
+                { (RuleExpressionsMetaData && showRuleComponent) && (
+                    <RuleConfigForm
+                        readonly={ getFieldDisabledStatus() }
+                        rule={ rule }
+                        setRule={ setRule }
+                        isHasRule={ isHasRule }
+                        setIsHasRule={ setIsHasRule }
+                    />
+                ) }
```

Comment:
No need of () around conditions here.

```suggestion
                { RuleExpressionsMetaData && showRuleComponent && (
                    <RuleConfigForm
                        readonly={ getFieldDisabledStatus() }
                        rule={ rule }
                        setRule={ setRule }
                        isHasRule={ isHasRule }
                        setIsHasRule={ setIsHasRule }
                    />
                ) }
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7486/commits/34a926448e3cea96642355bfe473951501c89739